chihacknight/state-by-state-decarbonization
# Total climate pollution chart & projection to 0 by 2050

## Overview
This PR adds the area chart to both the index and state detail pages.

Closes #17 

### Demo
[What does it take to decarbonize your state .pdf](https://github.com/chihacknight/state-by-state-decarbonization/files/8366505/What.does.it.take.to.decarbonize.your.state.pdf)



## Testing Instructions

This PR can be tested by loading the app and visually looking at the area chart on both the index and state pages.  Various state pages should be check to verify proper functioning.
